applications/asset_tracker/Kconfig

```diff
@@ -342,7 +342,7 @@ config USE_AT_HOST
 endmenu
 
 menu "Zephyr Kernel"
-source "$ZEPHYR_BASE/Kconfig.zephyr"
+source "Kconfig.zephyr"
```

Comment:
@rlubos : I don't see the harm in writing it like this, but @ulfalizer , is it expected that the previous mechanism would cause issues?

Reply:
Let me just paste how CI justifies this approach (and actually fails when former approach is used):
```
Redundant 'source "$(ZEPHYR_BASE)/Kconfig.zephyr" in 'applications/asset_tracker/Kconfig'. Just do 'source "Kconfig.zephyr"'
instead. The $srctree environment variable already points to the Zephyr root,
and all 'source's are relative to it.
```

Reply:
I see, it seems Ulf is very much aware of this then. :)